### PR TITLE
Simulation

### PR DESCRIPTION
**Pull request by 97gushan**
#13 
Simulate houses with turbines. Each able to sell its excess energy and buy more if needed

### CODE REVIEW

File: Simulator/simulation/simulator.js [hunk 1]
```diff
@@ -2,17 +2,63 @@ const fs = require('fs');
...
+		this.initHouses('config/house.json');
 
-		houses.forEach((house) => {
-			house.runSimulation();
+		this.houses.forEach((house) => {
+			setInterval(house.runSimulation, house.pollTime, house);
 		});
+		setInterval(this.calcPrice, 1000, this);
+	},
+	calcPrice: function(self) {
```
Comment (97gushan) [inline Simulator/simulation/simulator.js]: Should we have different polls for the houses and simulation or use the same?
Eg:
```javascript
runSimulation: function()
    this.houses.forEach((house) => {
        setInterval(house.runSimulation, house.pollTime, house);
    });
    setInterval(this.calcPrice, 1000, this);
},
```
Or:
```javascript
setInterval(this.runSimulation, 1000);

runSimulation: function()
    this.houses.forEach((house) => {
	 house.runSimulation();
    });
    this.calcPrice();
},
```

File: Simulator/simulation/simulator.js [hunk 2]
```diff
@@ -2,17 +2,71 @@ const fs = require('fs');
 const House = require('./house');
 
 Simulator = {
```
Comment (PandaFood) [inline Simulator/simulation/simulator.js:4]: Sometimes class sometimes object? We should choose one and stick to it

**Review by PandaFood — APPROVED** [review]

File: Simulator/simulation/simulator.js (hunk 2 above)
Comment (97gushan) [reply Simulator/simulation/simulator.js:4]: My thought was that we only want one instance of Simulator while we want several of house/windturbines thus the different uses

Comment (PandaFood) [reply Simulator/simulation/simulator.js:4]: 🤷

**Pull request merged**